.gitignore

```diff
@@ -42,4 +44,9 @@ protos/google/*
 satellite_*
 storagenode_*
 uplink_*
-*.svg
\ No newline at end of file
+*.svg
+big-upload-testfile
```

Comment:
The fix should be to create files in a temp directory and proper cleanup rather than ignoring them.

Reply:
I can create another ticket to do that. Does this comment also refer to the rest of the items in the .gitignore file?

Reply:
Yes, and I think some of them might have been fixed already.

Reply:
https://storjlabs.atlassian.net/browse/V3-432